src/utilities/utilities.ts

```diff
@@ -97,6 +97,15 @@ export async function execSwift(
     folderContext?: FolderContext
 ): Promise<{ stdout: string; stderr: string }> {
     const swift = getSwiftExecutable();
+
+    // add custom toolchain to path on Windows
+    if (process.platform === "win32" && configuration.path) {
+        options.env = {
+            Path: `${configuration.path};${process.env.Path}`,
+            ...options.env,
```

Comment:
If `options.env` already has a `Path` value in it, it will overwrite your new `Path` value.